python/servo/command_base.py

```diff
@@ -395,6 +399,79 @@ def get_binary_path(self, release, dev, android=False):
                                   " --release" if release else ""))
         sys.exit()
 
+    def get_nightly_binary_path(self, nightly_date):
+        if nightly_date is None:
+            return
+        if not nightly_date:
+            print(
+                "No nightly date has been provided although the --nightly or -n flag has been passed.")
+            sys.exit(1)
+        # Will alow us to fetch the relevant builds from the nightly repository
+        os_prefix = "linux"
+        if is_windows():
+            print("The nightly flag is not supported on windows yet.")
+            sys.exit(1)
+            os_prefix = "windows-msvc"
```

Comment:
why we change `os_prefix` after the program has exited?

Reply:
Good catch ! I was actually working on a multi platform solution, and then decided to split the merge requests. I'll remove it :)